pkg/runtime/runtime.go

```diff
@@ -215,6 +215,7 @@ func newDaprRuntime(ctx context.Context,
 		ActorsEnabled:   len(runtimeConfig.actorsService) > 0,
 		Registry:        runtimeConfig.registry,
 		ComponentStore:  compStore,
+		ComponentCb:     runtimeConfig.registry.ComponentsCallback(),
```

Comment:
is it true that at some point in the runtime initialization components callback is set to nil? I think I recall something like that

Reply:
It’s a pain to have a nil checker for this. If we can, let’s make sure this is always at least a noop func

Reply:
ah no, ignore me, its `a.appHealthReady` what its set to nil